tests/functional/encryption/test_encryption_keyrotation.py

```diff
@@ -236,6 +251,64 @@ def compare_old_keys_with_new_keys():
         log.info("Changing the keyrotation value to default.")
         noobaa_keyrotation.set_keyrotation_schedule("@weekly")
 
+    @pytest.mark.polarion_id("OCS-5963")
+    def test_bucket_checksum_with_noobaa_keyrotation(
+        self, mcg_obj_session, awscli_pod_session, bucket_factory, test_directory_setup
```

Comment:
Please consider using the non-session scoped fixtures of `mcg_obj` and and `awscli_pod`. 

While the pro of saving time on startup is tempting, the con is the potential for other tests to mess with these fixtures, which will mess with this test in turn. Personally, I'm now using the session-scoped only when they're absolutely needed, which is at mostly at upgrade TCs.

Also, be aware that the function-scoped `bucket_factory` is calling the function-scoped `mcg_obj` fixture:
https://github.com/red-hat-storage/ocs-ci/blob/master/tests/conftest.py/#L3050-L3051
Besides redundancy, this can cause difficult-to-debug issues.